dist/index.js

```diff
@@ -76,16 +76,16 @@ function movePositionByLines(code, numLines, position) {
     }
     if (opts.trimNewline) position++; // don't include prior newline
   } else if (numLines > 0) {
-      var numFollowingLines = numLines;
-      position++;
-      while (position < code.length && numFollowingLines > 0) {
-        if (code[position] === '\n') {
-          numFollowingLines--;
-        }
-        position++;
```

Comment:
I'm not sure why `dist/index.js` was updated, as `src/index.js` wasn't touched. Maybe different versions of `babel`?

I tried changing compile from this:

```
"compile": "babel -d dist/ src/",
```

to this:

```
"compile": "./node_modules/.bin/babel -d dist/ src/",
```

Because I suspected my global version of `babel` may have been different than the packaged one — no change.

We can look into this if it pops up again. In the mean time, I'm just going to remove this file from the commit